device/bgp/bgpsrtepolicynamesubtlv.yaml

```diff
@@ -0,0 +1,16 @@
+components:
+  schemas:
+    BgpSrte.policyNameSubTlv:
```

Comment:
nit, camelcase - BgpSrte.PolicyNameSubTlv - pointing out this but there are other instances of this.

Reply:
Done. Thanks